### DISCUSSION TITLE
Sending only sensor reading to cloud platform from EdgeX app service configurable

### DISCUSSION THREAD

**Discussion by sudhamani-hcl** [edgexfoundry/community#36] [General]
Hi Team,

Currently I am working on POC which demands visualization of sensor data in some cloud platform. To achieve this, I am using EdgeX app service and IBM Watson cloud platform. 
I could receive whole EdgeX event in the cloud platform. However my POC demands receiving of sensor reading alone.  
Is there any way, I can send only sensor reading to the cloud from EdgeX app service.

Thanks,
Sudhamani.

**Comment by cloudxxx8** [comment]
You can write your own App service based on App Function SDK to transform the data the format you want.
